### ISSUE TITLE
spree 3.3 and rails 5.1 support

### ISSUE THREAD

**Issue by Godweed** [spree-contrib/spree_globalize#34]
I am trying to install with spree 3.3 but getting below errors

Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.lock):
    activerecord (= 5.1.3)

  In Gemfile:
    spree_globalize was resolved to 3.2.0.alpha, which depends on
      globalize was resolved to 3.0.0, which depends on
        paper_trail (~> 2) was resolved to 2.6.3, which depends on
          activerecord (~> 3.0)

    rails (~> 5.1.3) was resolved to 5.1.3, which depends on
      activerecord (= 5.1.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.1.3)

  In Gemfile:
    spree_globalize was resolved to 3.2.0.alpha, which depends on
      globalize was resolved to 3.0.0, which depends on
        paper_trail (~> 2) was resolved to 2.0.0, which depends on
          rails (~> 3)

    spree (~> 3.3.0) was resolved to 3.3.0, which depends on
      spree_core (= 3.3.0) was resolved to 3.3.0, which depends on
        rails (~> 5.1.1)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

**Comment by roooodcastro** [comment]
Having the same issue.

I also want to ask, just started with Spree, doesn't this spree_globalize have released versions? I don't feel confortable at all referencing the master branch in production, yet there's no 'stable' branch nor any release tags?

**Comment by fxgit** [comment]
I had the same error solve it by adding to the Gemfile and run `bundle install`
```
gem 'globalize', github: 'globalize/globalize'
gem 'activemodel-serializers-xml'
```

**Comment by Godweed** [comment]
@fxgit thank you solved the bundle isssue

**Comment by ramesh2007** [comment]
@Godweed I didnt find any solution here, did you really solved your issue by adding with those gems

rails - 5.1.4
spree -3.3

Bundler could not find compatible versions for gem "spree":
  In Gemfile:
    spree (~> 3.3)

Could not find gem 'spree (~> 3.3)' in any of the sources.

Bundler could not find compatible versions for gem "spree_core":
  In Gemfile:
    spree (~> 3.3) was resolved to 3.3.0, which depends on
      spree_core (= 3.3.0)

    spree_redirects was resolved to 0.3.0, which depends on
      spree_core (~> 1.1)

**Comment by damianlegawiec** [comment]
Done!
